sdk/messaging/azservicebus/message.go

```diff
@@ -18,6 +17,11 @@ type ReceivedMessage struct {
 	// ApplicationProperties can be used to store custom metadata for a message.
 	ApplicationProperties map[string]interface{}
 
+	// Body corresponds to the first byte array in the Data section of an AMQP message.
```

Comment:
Will this extra info be useful for the customer?  i.e. how many will know (or need to know) about these details?  And could this ever change?  As long as we're ok with formalizing this contract.

Also, the change description implies that a `nil` body _might_ be an error.  Am I reading too much into that?  I know that AMQP allows sending messages with no body, does SB support the same?  I want to make sure we don't have some ambiguous situation when the body is nil.

Reply:
I think the key here is that only the user knows what a nil Body means. We can tell them we couldn't convert it, but the fact that they expected some payload but got nil tells them something is wrong (which is why it felt okay to just convert this back to a field).

If Service Bus didn't support raw AMQP messages then it'd be unneeded, but since we do it makes sense to document where this simplified envelope doesn't work (IMO).

When we add in raw AMQP message support they'll also be able to do something about it but for now it's just informational.

Reply:
Removed.

Reply:
Whoops, one second.

Reply:
Now it's removed.